lib/validation.ex

```diff
@@ -26,6 +26,8 @@ defmodule Uploadex.Validation do
     end
   end
 
+  defp all_extensions_accepted?(_files, :any), do: true
```

Comment:
Is this necessary? We are already handling this in `def validate_extensions(_files, :any), do: :ok`

Reply:
Yes, because of the last breaking change in the lib